plugins/connection/lxc.py

```diff
@@ -86,7 +86,7 @@ def _communicate(self, pid, in_data, stdin, stdout, stderr):
             write_fds = []
         while len(read_fds) > 0 or len(write_fds) > 0:
             try:
-                ready_reads, ready_writes, _ = select.select(read_fds, write_fds, [])
+                ready_reads, ready_writes, unused_var = select.select(read_fds, write_fds, [])
```

Comment:
Please always use `dummy` as the placeholder (https://docs.ansible.com/ansible/latest/dev_guide/testing/sanity/no-underscore-variable.html).